dotnet/samples/Demos/ModelContextProtocol/McpClientUtils.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using McpDotNet.Client;
+using McpDotNet.Configuration;
+using McpDotNet.Protocol.Types;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.SemanticKernel;
+
+namespace ModelContextProtocol;
+internal static class McpClientUtils
+{
+    internal static async Task<IMcpClient> GetSimpleToolsAsync()
+    {
+        McpClientOptions options = new()
+        {
+            ClientInfo = new() { Name = "GitHub", Version = "1.0.0" }
+        };
+
+        var config = new McpServerConfig
+        {
+            Id = "github",
+            Name = "GitHub",
+            TransportType = "stdio",
+            TransportOptions = new Dictionary<string, string>
+            {
+                ["command"] = "npx",
+                ["arguments"] = "-y @modelcontextprotocol/server-github",
+            }
+        };
+
+        var factory = new McpClientFactory(
+            [config],
+            options,
+            NullLoggerFactory.Instance
+        );
+
+        return await factory.GetClientAsync(config.Id).ConfigureAwait(false);
+    }
+
+    internal static async Task<IEnumerable<KernelFunction>> MapToFunctionsAsync(this IMcpClient mcpClient)
+    {
+        var tools = await mcpClient.ListToolsAsync().ConfigureAwait(false);
+        return tools.Tools.Select(t => t.ToKernelFunction(mcpClient)).ToList();
+    }
```

Comment:
suggest updating it to `McpClientExtensions` and convert the `GetSimpleToolsAsync` and anyu other Non-Extenssions to be as well.

```suggestion
    internal static async Task<IEnumerable<KernelFunction>> MapToFunctionsAsync(this IMcpClient mcpClient)
    {
        var tools = await mcpClient.ListToolsAsync().ConfigureAwait(false);
        return tools.Tools.Select(t => t.ToKernelFunction(mcpClient)).ToList();
    }
```